virttest/qemu_vm.py

```diff
@@ -3669,6 +3670,15 @@ def migrate(self, timeout=virt_vm.BaseVM.MIGRATE_TIMEOUT, protocol="tcp",
             if offline is True:
                 self.monitor.cmd("stop")
 
+            if migrate_capabilities:
+                for key, value in migrate_capabilities.items():
+                    state = value == "on"
+                    self.monitor.set_migrate_capability(state, key)
+                    s = self.monitor.get_migrate_capability(key)
+                    if s != state:
+                        raise exceptions.TestError("Cannot get the correct "
+                                                   "migrate capability value.")
```

Comment:
1. "Failed to set migrate capability" or "Failed to turn on migrate capability" make more clear.
2. you will set all capabilities? xbzrle, auto-converge ... at the same time?